src/items/FileItem.tsx

```diff
@@ -120,9 +109,21 @@ const FileItem: FC<FileItemProps> = ({
   }
 
   const getComponent = (): JSX.Element => {
+    const fileExtra =
+      item.type === ItemType.LOCAL_FILE ? getFileExtra(item.extra) : undefined;
+    const s3FileExtra =
+      item.type === ItemType.S3_FILE ? getS3FileExtra(item.extra) : undefined;
+    const { mimetype, name: originalFileName } = {
+      ...fileExtra,
+      ...s3FileExtra,
+    };
+    const altText =
+      item.type === ItemType.S3_FILE || item.type === ItemType.LOCAL_FILE
+        ? item.settings.altText || item.name
```

Comment:
To me it feels strange to have to pull that property out of the settings, when we already pull the mimetype and originalFileName from the extras ... @MalinSvenberg